daprdocs/content/en/operations/support/support-preview-features.md

```diff
@@ -15,8 +15,7 @@ For CLI there is no explicit opt-in, just the version that this was first made a
 ## Current preview features
 | Feature | Description | Setting | Documentation | Version introduced |
 | ---------- |-------------|---------|---------------|-----------------|
-| **Partition actor reminders** | Allows actor reminders to be partitioned across multiple keys in the underlying statestore in order to improve scale and performance. | `Actor.TypeMetadata` | [How-To: Partition Actor Reminders]({{< ref "howto-actors.md#partitioning-reminders" >}}) | v1.4 |
-| **Pub/Sub routing** | Allow the use of expressions to route cloud events to different URIs/paths and event handlers in your application. | `PubSub.Routing` | [How-To: Publish a message and subscribe to a topic]({{<ref howto-route-messages>}}) | v1.7 |
+| **Pub/Sub routing** | Allow the use of expressions to route cloud events to different URIs/paths and event handlers in your application. | `PubSub.Routing` | [How-To: Publish a message and subscribe to a topic]({{<ref howto-route-messages>}}) | v1.4 |
 | **ARM64 Mac Support** | Dapr CLI, sidecar, and Dashboard are now natively compiled for ARM64 Macs, along with Dapr CLI installation via Homebrew. | N/A | [Install the Dapr CLI]({{<ref install-dapr-cli>}}) | v1.5 |
```

Comment:
Given that you are updating this file already, we agreed that this ARM64 support for Mac was also no longer needed in preview. Can you remove this also?
```suggestion
```